pyproject.toml

```diff
@@ -217,6 +218,9 @@ filterwarnings = [
   'ignore:.*The distutils package is deprecated and slated for removal in Python 3.12:DeprecationWarning:pyspark',
   'ignore:.*distutils Version classes are deprecated. Use packaging.version instead.*:DeprecationWarning:pyspark',
   'ignore:.*is_datetime64tz_dtype is deprecated and will be removed in a future version.*:DeprecationWarning:pyspark',
+  'ignore:.*No Partition Defined for Window operation! Moving all data to a single partition.*:UserWarning:pyspark',
+  # TODO(FBruzzesi): This is raised in `hist_test.py` from pandas??? How does spark-connect affect/changes pandas behavior?
+  'ignore:.*invalid value encountered in cast.*:RuntimeWarning',
```

Comment:
This other warning is just WTF 🤔

Reply:
I can reproduce this with _local_ session with pyspark and old pandas/python versions